### PR TITLE
[fix] Add permission to view org to operator

### LINKED ISSUE #276: [bug] Add permission to view organizations to operator
Add permission to view organizations to operator (otherwise they cannot see their own organization in select2 autocomplete fields).

### PR DESCRIPTION
**Pull request by Aryamanz29**
Fixes #276

### CODE REVIEW

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
@Aryamanz29 the tests are failing, please always run tests locally before opening PRs.

File: openwisp_users/tests/test_api/test_api.py [hunk 1]
```diff
@@ -215,12 +215,14 @@ def test_create_group_list_api(self):
...
         path = reverse('users:group_detail', args='1')
-        with self.assertNumQueries(3):
+        with self.assertNumQueries(4):
             r = self.client.get(path)
         self.assertEqual(r.status_code, 200)
         self.assertEqual(r.data['id'], 1)
         self.assertEqual(r.data['name'], 'Operator')
-        self.assertEqual(r.data['permissions'], [])
+        self.assertEqual(
+            r.data['permissions'], ['40: organization | Can view organization']
```
Comment (codesankalp) [inline openwisp_users/tests/test_api/test_api.py:224]: @Aryamanz29 You can use 
```
permission = Permission.objects.filter(codename='view_organization')
self.assertEqual(
            r.data['permissions'], [f'{permission.id}: organization | {permission.name}']
)
```

Comment (codesankalp) [reply openwisp_users/tests/test_api/test_api.py:224]: Or you can use `assertIn` also.

**Review by nemesisdesign — APPROVED** [review]
Thanks!

**Pull request merged**